Doc/library/dataclasses.rst

```diff
@@ -97,7 +97,17 @@ Module contents
      the name and repr of each field, in the order they are defined in
      the class.  Fields that are marked as being excluded from the repr
      are not included.  For example:
-     ``InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)``.
+
+     .. doctest::
+
+        >>> from dataclasses import dataclass, field
+        >>> @dataclass(repr=False)
+        ... class InventoryItem:
+        ...     name: str
+        ...     unit_price: float
+        ...     quantity_on_hand: int
+        >>> InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10) #doctest:+ELLIPSIS
```

Comment:
```suggestion
        ...     quantity_on_hand: int
        ...
        >>> InventoryItem(name='widget', unit_price=3.0, quantity_on_hand=10)  # doctest: +ELLIPSIS
```

Just to be consistent with the REPL itself and the rest of the documentation in general.

Reply:
(When you press ENTER, you will have this additional empty line with just `...`)

Reply:
This is my first time with python and docs, so your comments helps a lot, thank you.